kong/db/strategies/cassandra/connector.lua

```diff
@@ -263,6 +264,34 @@ function CassandraConnector:reset()
   return true
 end
 
+function CassandraConnector:init_worker()
+  if self.refresh_frequency > 0 then
```

Comment:
how can we disable the auto update?

Reply:
I do not understand what you mean, if the value is 0 then this should not execute. Could you explain what you mean further?

Reply:
Please log a message if it is disabled and document that 0 is disabled

Reply:
and if algo is if it enabled log the `refresh_frequency` :)